_plays/08.md

```diff
@@ -4,10 +4,12 @@ title: Make it easy to use
 ---
 
 ### Checklist
-1.  Provide reference implementation.  Show how to use the API.
-2.  Write modifiable exampls.
-3.  Even create cliend-side wrappers with use, if you have the resources.  This brings the service even closer to the end-use.
+1.  Provide reference implementation.  
```

Comment:
Thought it was better with "Show how to use the API". Reference implementation may not have clear meaning. Or can we share examples of where this is done well?